### ISSUE TITLE
Screen reader announce irrelevant information as “[F]” for “Go full screen” and “Exit full screen” button. 

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#27388]
### Describe the bug

Screen reader users will be affected if screen reader announce irrelevant information along with the controls. 

### Reproduction link

https://mgt.dev/?path=/docs/overview--docs

### Reproduction steps

1. Open the URL  https://mgt.dev/?path=/docs/overview--docs

2. Turn on Narrator using ‘Ctrl+win+enter’ keys. 

3. The ‘Microsoft Graph Toolkit Playground' Overview page will get displayed. 

4. Navigate to the ‘Go Full screen’ button. 

5. Observe the issue with the screen reader announcement. 

https://github.com/storybookjs/storybook/assets/93735775/cdc0fc2b-692e-45ed-aa2c-5f7dde46763f

### System

```bash
Test Environment: 
OS Build: Windows 11  
Version: 24H2 (OS Build 26058.1400)  
Browser: Edge dev  
Browser Version 123.0.2400.1 (Official build) dev (64-bit)
```

### Additional context

**Actual result:**
Screen reader announce irrelevant information as “[F]” for “Go full screen” and “Exit full screen” button. 

**Expected Results:**
Screen reader should not announce irrelevant information as “[F]” for “Go full screen” and “Exit full screen” button.

**Comment by Sranjan0208** [comment]
I inspected the page and from what I see the problem is within the button element. 
![image](https://github.com/storybookjs/storybook/assets/90925047/d67baffb-1048-41b2-9a62-a70cad20b86d)
As you can see the button element has an attribute title which says "Go Full Screen [F]", so the screen reader is reading that. 
The fix is to provide an additional attribute called `aria-label: "Go Full Screen"`. This attribute will explicitly define what the screen reader should announce.

**Comment by msftedad** [comment]
Hi @Sranjan0208, Any update on the fix?

**Comment by nick-fallon** [comment]
This has, apparently, been fixed.

<img width="1431" alt="Screen Shot 2024-06-09 at 1 31 46 AM" src="https://github.com/storybookjs/storybook/assets/110064775/2b166616-8a10-43ad-a923-9cdda62581e7">

**Comment by msftedad** [comment]
@nick-fallon, Thanks for the information.We have closed our issue.Thanks

**Comment by rishabhrthr001** [comment]
@msftedad if this issue is still open can i work on it

**Comment by msftedad** [comment]
@rishabhrthr001, Thanks for following up, This issue is fixed now.

**Comment by akhil02hero** [comment]
@msftedad if the issue is fixed can you close the issue, it helps the contributor or anyone who is willing to contribute know that issue is no more required, just a suggestion from my end.

**Comment by msftedad** [comment]
Hi @akhil02hero, closing this issue as issue has been fixed. attached screenshot for Reference.
![Image](https://github.com/user-attachments/assets/692f9140-b3bc-4f74-a738-df90a695cc8a)
